spec/swagger/swagger_doc_v1.yaml

```diff
@@ -3427,6 +3427,91 @@
             application/vnd.api+json:
               schema:
                 "$ref": "../v1/get_ethnicities_responses.yaml#/415"
+  "/reference/frameworks":
+    get:
+      summary: Returns a list of assessment frameworks
+      tags:
+        - Frameworks
+      consumes:
+        - application/vnd.api+json
+      parameters:
+        - "$ref": "../v2/accept_type_parameter.yaml#/Accept"
+        - "$ref": "../v1/content_type_parameter.yaml#/ContentType"
+      responses:
+        "200":
+          description: success
+          content:
+            application/vnd.api+json:
+              schema:
+                "$ref": "../v1/get_frameworks_responses.yaml#/200"
```

Comment:
This is returning relationships with questions, maybe its better if this as empty as we won't allow includes on the list endpoint

Reply:
Okay, so this means I will need to duplicate `Framework` definition to have a variant of that without the associated questions.